duneana/AnaTree/AnalysisTree_module.cc

```diff
@@ -4611,6 +4760,12 @@ void dune::AnalysisTree::analyze(const art::Event& evt)
       else
         PFParticleData.pfp_isTrack[i] = 0;
 
+      const larpandoraobj::PFParticleMetadata metaData = *((theMetadataAssns.at(pfparticlelist[i]->Self())).at(0));
```

Comment:
Is it possible for any of these `at(int)` calls to segfault?

Reply:
Done, I added a check for for valid and size